js/widgets.js

```diff
@@ -1,108 +1,172 @@
-Array.range = function(n) {
-    // Array.range(5) --> [0,1,2,3,4]
-    return Array.apply(null, Array(n)).map((x, i) => i);
-};
+/* This file is using http://standardjs.com/ */
```

Comment:
Even if it may be a good/best practice, It adds another convention. We are still migrating files to ES6 using the eslint rules :/